src/lib.rs

```diff
@@ -603,48 +621,68 @@ pub trait Rng {
 }
 
 impl<'a, R: ?Sized> Rng for &'a mut R where R: Rng {
+    #[inline]
     fn next_u32(&mut self) -> u32 {
         (**self).next_u32()
     }
 
+    #[inline]
     fn next_u64(&mut self) -> u64 {
         (**self).next_u64()
     }
 
+    #[inline]
     fn next_f32(&mut self) -> f32 {
         (**self).next_f32()
     }
 
+    #[inline]
     fn next_f64(&mut self) -> f64 {
         (**self).next_f64()
     }
 
+    #[inline]
     fn fill_bytes(&mut self, dest: &mut [u8]) {
         (**self).fill_bytes(dest)
     }
+    
+    #[inline]
+    fn try_fill_bytes(&mut self, dest: &mut [u8]) -> Result<(), Error> {
+        (**self).try_fill_bytes(dest)
+    }
 }
 
 #[cfg(feature="std")]
```

Comment:
Not part of this PR, but should this also be available with the `alloc` feature?

Reply:
With that I mean `impl<R: ?Sized> Rng for Box<R> where R: Rng {`

Reply:
Ah. Probably. Make a separate PR?

Reply:
Okay. But I should probably wait until this PR has landed because they will conflict?